imgui-SFML.cpp

```diff
@@ -25,6 +25,14 @@
 #include <memory>
 #include <vector>
 
+#ifdef SFML_VERSION_MAJOR >= 3
```

Comment:
```suggestion
#if SFML_VERSION_MAJOR >= 3
```

I think it needs to be written like this.
currently there is an error.

```
/Users/runner/work/imgui-sfml/imgui-sfml/imgui-sfml/imgui-SFML.cpp:28:27: warning: extra tokens at end of #ifdef directive [-Wextra-tokens]
#ifdef SFML_VERSION_MAJOR >= 3
                          ^
                          //
/Users/runner/work/imgui-sfml/imgui-sfml/imgui-sfml/imgui-SFML.cpp:365:10: error: no member named 'Apostrophe' in 
```

Reply:
thanks indeed !